frontend/src/app/aethel/aethel.component.html

```diff
@@ -0,0 +1,121 @@
+<h2 class="title is-2" i18n>Æthel</h2>
+
+<p class="field" i18n>
+    Welcome to Æthel. Search for a word or lemma in the input field below to
+    start.
+</p>
+
+<form class="form">
+    <div class="field">
+        <label for="aethel-input" class="label">Search:</label>
+        <div class="aethel-input-container">
+            <input
+                id="aethel-input"
+                class="input"
+                [class.is-danger]="aethelInput.touched && aethelInput.invalid"
+                type="text"
+                [formControl]="aethelInput"
+                (keydown.enter)="$event.preventDefault(); search()"
+                placeholder="Enter a word or lemma..."
+                placeholder-i18n
+            />
+            <button
+                type="button"
+                class="button is-primary"
+                [class.is-loading]="loading$ | async"
+                (click)="search()"
+            >
+                <span i18n>Search</span>
+            </button>
+        </div>
+        @if (aethelInput.touched && aethelInput.invalid) {
```

Comment:
is `@if` new? and/or better than ngIf?

Reply:
New since Angular 16. It behaves the same as `@ngIf`, but it looks a bit cleaner in the template (I think).